server/handlers/project_command_output_handler.go

```diff
@@ -0,0 +1,134 @@
+package handlers
+
+import (
+	"sync"
+
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
+)
+
+type DefaultProjectCommandOutputHandler struct {
+	// this is TerraformOutputChan
+	ProjectCmdOutput chan *models.ProjectCmdOutputLine
+	// this logBuffers
+	projectOutputBuffers map[string][]string
+	// this is wsChans
+	controllerBuffers map[string]map[chan string]bool
```

Comment:
Let's rename this into `receiverBuffers`
```suggestion
	receiverBuffers map[string]map[chan string]bool
```